### ISSUE TITLE
Fixed button layout on welcome

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network#323]
Continuing the work #288, let's find a way to have fixed layout buttons on the welcome screen. I'm concerned that a ton of users will bounce if they only see a wall of text.

### desktop

desktop is good and resolved. 

![welcome-desktop](https://cloud.githubusercontent.com/assets/53411/25138200/e56f58b4-2427-11e7-880f-7ef01f91e8d6.jpg)

### mobile

side-by-side, they fit well. the style could be better, but many fewer people will be likely to bounce. 

![welcome-mobile](https://cloud.githubusercontent.com/assets/53411/25138199/e56b5f0c-2427-11e7-9289-011802051e18.jpg)

**Comment by sabrinang** [comment]
<img width="1288" alt="image" src="https://cloud.githubusercontent.com/assets/2492510/25151918/2dd31406-243c-11e7-8e9e-fa3779b30950.png">
With the amount of words and type size "join the network" would fall on two lines making bigger buttons. Included a bit of a fade to hint there is more text but it might not be obvious.

1 - using grid margins balances can help balance out the buttons
2 - side-by-side style you have above, no margins

@xmatthewx feedback?

**Comment by xmatthewx** [comment]
I like the button margin. Much better. Instead of fade (element could block your thumb swipe), let's try a white background with single pixel shadow like in desktop above.

**Comment by sabrinang** [comment]
<img width="662" alt="fixed buttons - iteration with divider" src="https://cloud.githubusercontent.com/assets/2492510/25155957/a97e3c2e-244c-11e7-9184-bb901dad1369.png">

**Comment by xmatthewx** [comment]
@gvn – we'd like to put the Welcome page's buttons in a fixed container at bottom of the screen. It's up to you whether you tack this on now, and revisit this next week. I made this a separate ticket to unblock your build. 

Rationale: We worry that a ton of visitors will bounce without accessible buttons. We hope they read info first, but this gives them some choice.

[Desktop mockup](https://github.com/mozilla/network/issues/323#issue-222451229) is in first comment. [Mobile mockup](https://github.com/mozilla/network/issues/323#issuecomment-295004433) is comment right above.

@sabrinang can pass along specs or sketch if needed.

**Comment by gvn** [comment]
I'll see how hard it is to add now. I'd recommend just solving the root problem through copy editing though.

**Comment by gvn** [comment]
@sabrinang If you're done with this work can you remove your assignment?

**Comment by xmatthewx** [comment]
@gvn - we need a bit more padding at the bottom on mobile. Do you want me to file a new ticket?

<img width="336" alt="screen shot 2017-04-21 at 2 46 36 pm" src="https://cloud.githubusercontent.com/assets/53411/25293888/b3cfb5ee-26aa-11e7-862a-27667d9839f0.png">

**Comment by gvn** [comment]
Sure
